tldr.py

```diff
@@ -504,6 +508,40 @@ def main() -> None:
                 with open(command, encoding='utf-8') as open_file:
                     output(open_file.read().encode('utf-8').splitlines(),
                            plain=options.markdown)
+    elif options.search:
+        command = '-'.join(options.command)
+        get_commands(options.platform)
+        page = ''
+        maxprob = 0
+        searchquery = options.search.split(' ')
+        for i in get_commands(options.platform):
+            if i.startswith(command):
+                result = get_page(
+                        i,
+                        options.source,
+                        options.platform,
+                        options.language
+                    )
+                for word in searchquery:
+                    prob = 0
+                    for line in result:
+                        line = line.decode('utf-8')
+                        if word in line:
+                            if (line.startswith('-') or line.startswith('>')):
+                                prob += 1
```

Comment:
```suggestion
                        if word in line and (line.startswith('-') or line.startswith('>')):
                            prob += 1
```

Reply:
I actually wrote it like this initially but flake8 check failed since it exceeded the character limit for a line

Reply:
You're right, here's the modified if statement that satisfies flake8, while keeping it as one if statement:
```
                        if word in line and \
                                (line.startswith('-') or line.startswith('>')):
```